[Docs] Added docs on how to handle a large response when using FlyteRemote.sync (#2932)
flyteorg/flyte: rsts/community/troubleshoot.rst

- Flyte uses the format ``executionid-node-id-attempt`` from the node to assign a name to a Kubernetes pod or downstream resource.
- But if this is an invalid name for a Kubernetes pod, Flyte assigns a valid name of random characters instead.

Troubles with handling large responses in ``FlyteRemote.sync`` 
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

- ``Received message larger than max (xxx vs. 4194304)`` usually crops up when the message size is too large.

> - To fix this, edit the flyte-admin-base-config config map to increase ``maxMessageSizeBytes`` value.

Reviewer: Can you also add the kubectl command mentioned in the slack conversation?

Author: OK, I'll do that, but I'll also add the `^^^` to fix the error, I'm sorry I didn't notice that before

Reviewer: There's a command mentioned in the conversation I included in the PR description, right? Can you include it?

Reviewer: You can say *** is how you can edit the flyte-admin-base-config config map.

Author: Yup I include it

- Flyte uses the format ``executionid-node-id-attempt`` from the node to assign a name to a Kubernetes pod or downstream resource.
- But if this is an invalid name for a Kubernetes pod, Flyte assigns a valid name of random characters instead.

Troubles with handling large responses in ``FlyteRemote.sync`` 
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

- ``Received message larger than max (xxx vs. 4194304)`` usually crops up when the message size is too large.
- To fix this, edit the flyte-admin-base-config config map to increase ``maxMessageSizeBytes`` value.

> This is how you can edit the flyte-admin-base-config config map 

Reviewer: Please remove these lines.

Author: done

> - Edit the ``flyte-admin-base-config`` config map using the command ``kubectl edit cm flyte-admin-base-config -n flyte`` to increase the ``maxMessageSizeBytes`` value. Refer to the :ref:`flyte: <troubleshoot>` in case you've queries about the command's usage.

Reviewer: ```suggestion
- To fix this, edit the ``flyte-admin-base-config`` config map using the command ``kubectl edit cm flyte-admin-base-config -n flyte`` to increase the ``maxMessageSizeBytes`` value. Refer to the :ref:`flyte: <troubleshoot>` in case you've queries about the command's usage.
```

- Flyte uses the format ``executionid-node-id-attempt`` from the node to assign a name to a Kubernetes pod or downstream resource.
- But if this is an invalid name for a Kubernetes pod, Flyte assigns a valid name of random characters instead.

> Troubles With ``FlyteRemote.sync`` handling large response when using FlyteRemote.sync
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Reviewer: ```suggestion
Troubles with handling large responses in ``FlyteRemote.sync`` 
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

> - Usually occurs when trying to sync execution using sync() and the elements or nodes exceed the max limit previously set.
> - The log displays ``Received message larger than max``
> - The log ends with the grpc message ``grpc_message:"Received message larger than max ( vs. 4194304)"``

Reviewer: ```suggestion
- ``Received message larger than max (xxx vs. 4194304)`` usually crops up when the message size is too large.
```

> 
> How To Fix?
> """""""""""""""
> - In the ``config`` file of flyte-admin, increase the value of ``MaxMessageSizeBytes`` variable accordingly.

Reviewer: ```suggestion
- To fix this, edit the flyte-admin-base-config config map to increase ``maxMessageSizeBytes`` value.
```

Author: Done, I updated it..
